### PR TITLE
Rewrite intro of alias named signature

### PR DESCRIPTION
**Pull request by W4anD0eR96**
maybe puzzle user: https://irclog.perlgeek.de/perl6/2017-10-10#i_15282583
add example

### CODE REVIEW

File: doc/Type/Signature.pod6 [hunk 1]
```diff
@@ -551,8 +551,12 @@ variable name:
 
 Aliases are also possible that way:
 
-    sub ( :color(:colour($c)) ) { }    # 'color' and 'colour' are both OK
-    sub ( :color(:$colour) ) { }       # same API for the caller
+    sub alias-named(:color(:$colour), :type(:class(:$kind))) { say $colour ~ " " ~ $kind }
+    alias-named(color => "red", type => "A");     # both name can be use
```
Comment (MasterDuke17) [inline doc/Type/Signature.pod6]: This should be "both names can be used".

File: doc/Type/Signature.pod6 [hunk 2]
```diff
@@ -551,8 +551,12 @@ variable name:
...
+    alias-named(colour => "green", type => "B");  # more than two name is ok
```
Comment (MasterDuke17) [inline doc/Type/Signature.pod6]: And this should be "more than two names are ok".

File: doc/Type/Signature.pod6 [hunk 3]
```diff
@@ -551,8 +551,12 @@ variable name:
...
+    alias-named(color => "white", class => "C");  # every alias is independent
+
+More use of alias can be found in L<sub MAIN|/language/functions#sub_MAIN>
```
Comment (MasterDuke17) [inline doc/Type/Signature.pod6]: "aliases"

**Comment by W4anD0eR96** [comment]
@MasterDuke17 all are fixed, sorry for poor English

**Comment by Tyil** [comment]
No problem at all, thanks for the PR!

**Pull request merged**